src/lib/operators/aggregate_sort.cpp

```diff
@@ -256,6 +256,69 @@ void AggregateSort::_set_and_write_aggregate_value(
   }
 }
 
+Segments AggregateSort::_get_segments_of_chunk(const std::shared_ptr<const Table>& input_table, const ChunkID chunk_id) {
+  Segments segments{};
+  const auto column_count = input_table->column_count();
+  segments.reserve(column_count);
+  const auto chunk = input_table->get_chunk(chunk_id);
+  for (auto column_id = ColumnID{0}; column_id < column_count; ++column_id) {
+    segments.emplace_back(chunk->get_segment(column_id));
```

Comment:
Missing nullptr check

Reply:
Added. Wondering why table_scan.cpp:99 does not need to check for that. Can you elaborate a bit because `get_chunk` does not really help me.

Reply:
It does in the next line:

```
    Assert(chunk_in, "Physically deleted chunk should not reach this point, see get_chunk / #1686.");
```

Physically deleted chunks are removed by GetTable, but we should still check that `chunk` is not a `nullptr`.

Reply:
I am blind.

Reply:
Added.